packages/components/src/unit-control/index.native.js

```diff
@@ -149,6 +161,7 @@ function UnitControl( {
 					minimumValue={ min }
 					maximumValue={ max }
 					value={ value }
+					step={ unitStep }
```

Comment:
Is this intentional? Or should it be:

```suggestion
					step={ step }
```